publicdb/status_display/views.py

```diff
@@ -523,8 +523,8 @@ def station_latest(request, station_number):
                                   station=station)
                           .latest())
 
-    down, problem, up = status_lists()
-    status = get_station_status(station.number, down, problem, up)
+    station_status = StationStatus()
```

Comment:
Seems a bit overkill to get status for all stations only to determine the status of a single station.

Maybe some caching of the status could help.. in most cases they could be determined after a run of hisparcupdate, since that updates all summaries, but a shorter cache would be preferable, just in case Pcs are deactivated, or other manual changes are made..

Reply:
In principle, I agree. However, this was previously the case as well so I don't currently consider this a problem. We could add a class method on StationStatus which only checks a single station?

Reply:
It is not a problem, unless it is slow.

Reply:
It seems to be rather quick. Only tens of milliseconds per query.